src/main/java/gov/nasa/pds/tools/validate/rule/pds4/FileReferenceValidationRule.java

```diff
@@ -220,16 +231,45 @@ private boolean validate(DocumentInfo xml) {
               passFlag = false;
               continue;
             }
+
+            boolean pdfValidateFlag = false;
+            PDFUtil pdfUtil = null;
             for (TinyNodeImpl child : children) {
               if ("file_name".equals(child.getLocalPart())) {
                 name = child.getStringValue();
-LOG.debug("FileReferenceValidationRule:validate:name {}",name);
+                LOG.debug("FileReferenceValidationRule:validate:name {}",name);
+
+                // If the name is a PDF file, validate it.
+                if (name.endsWith(PDF_ENDS_WITH_PATTERN)) {
+                    if (pdfUtil == null) {
+                        pdfUtil = new PDFUtil(getTarget());
+                    }
+                    pdfValidateFlag = pdfUtil.validateFileStandardConformity(name);
+                    // Report a  warning if the PDF file is not PDF/A compliant.
+                    if (!pdfValidateFlag) {
+                        URL urlRef = null;
+                        if (!directory.isEmpty()) {
+                          urlRef = new URL(parent, directory + "/" + name);
+                        } else {
+                          urlRef = new URL(parent, name);
+                        }
+                        ProblemDefinition def = new ProblemDefinition(
+                            ExceptionType.WARNING, ProblemType.NON_PDFA_FILE,
+                            urlRef.toString() + " is not valid PDF/A file or does not exist");
+                        problems.add(new ValidationProblem(def, target,
+                            fileObject.getLineNumber(), -1));
+                        passFlag = false;
+                    }
+                }
               } else if ("md5_checksum".equals(child.getLocalPart())) {
                 checksum = child.getStringValue();
+LOG.debug("FileReferenceValidationRule:validate:checksum {}",checksum);
```

Comment:
@qchaupds very nitpicky here, but can we fix this whitespace if we want to keep these log messages in this method?